src/scrapers/tools/geocoder.py

```diff
@@ -10,29 +10,32 @@
 from models.initiatives import InitiativeImport
 
 
+def get_logger() -> logging.Logger:
+    return logging.getLogger(__name__)
+
+
 class Geocoder:
 
     def __init__(self):
         self.geo_locator = Nominatim(user_agent="code-for-nl-covid-19")
 
-    def get_logger(self) -> logging.Logger:
-        return logging.getLogger(__name__)
-
     def batch(self, feature_type=None):
-        logger = self.get_logger()
+        logger = get_logger()
         db = Db()
         # Default concat function to sqlite
-        concat_func = func.group_concat(InitiativeImport.id.distinct()).label('id_array')
+        concat_func = func.group_concat(InitiativeImport.id.distinct()).label("id_array")
 
-        if db.session.bind.driver != 'pysqlite':
+        if db.session.bind.driver != "pysqlite":
             # Assume postgres
-            concat_func = func.array_agg(InitiativeImport.id, type_=ARRAY(Integer)).label('id_string')
+            concat_func = func.array_agg(InitiativeImport.id, type_=ARRAY(Integer)).label("id_string")
 
         location_set = db.session.query(
```

Comment:
Hmmm, zouden we ook niet moeten filteren op items die nog geen osm_address hebben? Ik dacht dat de oude code ook al zoiets deed, maar ik kan dat nu helemaal niet vinden. 🙄